### PR TITLE
📦 scxa-marker-gene-heatmap 💬 Bugfix #181594058 – Fix cell type marker gene heatmap endpoints

### PR DESCRIPTION
**Pull request by lingyun1010**
Three things fixed here:
1. Refactor the cell type marker gene heatmap data enpoint by removing cell type value parameters. This feature should go with the new [web-sc PR](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/257) 
2. Populate the cell type dropdown options from heatmap data
3. Load the heatmap component based on the cluster type, i.e. cell type or clusters.

### CODE REVIEW

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
As before, colours isn’t the right word. Please use an accurate variable name that represents its purpose.

File: packages/scxa-marker-gene-heatmap/html/ClustersViewDemo.js [hunk 1]
```diff
@@ -61,20 +59,19 @@ class Demo extends React.Component {
...
-                    //sort() is very important here! Otherwise, it will fetch though the values in list are not changed
-                    .search({cellGroupType: this.state.selectedColourBy, cellType: this.state.cellTypes.sort() })
+                    .search({cellGroupType: this.state.selectedColourBy})
                     .toString() :
                 URI(`json/experiments/${this.state.experimentAccession}/marker-genes/clusters`)
                   .search({k: this.state.selectedColourBy})
                   .toString()
           }
-          host={`https://wwwdev.ebi.ac.uk/gxa/sc/`}
+          host={`http://localhost:8080/gxa/sc/`}
```
Comment (upendrakumbham) [inline packages/scxa-marker-gene-heatmap/html/ClustersViewDemo.js:69]: is this `localhost` URL expected here?

Comment (upendrakumbham) [reply packages/scxa-marker-gene-heatmap/html/ClustersViewDemo.js]: Good to see.

**Review by upendrakumbham — DISMISSED** [review]
All looks good to me!

**Review by upendrakumbham — APPROVED** [review]
ALGTM!

**Review by alfonsomunozpomer — APPROVED** [review]
All fine!

**Pull request merged**